images/update-dbg/dbg-go/pkg/autogenerate/constants.go

```diff
@@ -0,0 +1,28 @@
+package autogenerate
+
+const (
+	urlArchTemplate = "https://raw.githubusercontent.com/falcosecurity/kernel-crawler/kernels/{{ .Architecture }}/list.json"
+	urlLastDistro   = "https://raw.githubusercontent.com/falcosecurity/kernel-crawler/kernels/last_run_distro.txt"
+	configPathFmt   = "%s/driverkit/config/%s/%s/%s" // Eg: repo-root/driverkit/config/5.0.1+driver/x86_64/centos_5.14.0-325.el9.x86_64_1.yaml
+	outputPathFmt   = "output/%s/%s/%s_%s_%s_%s"     // Eg: output/5.0.1+driver/x86_64/falco_centos_5.14.0-325.el9.x86_64_1.{ko,o}
+)
+
+var (
+	// Please add new supported build-new-drivers structures here,
+	// so that the utility starts building configs for them.
+	// Fields must have the same name used in kernel-crawler json keys.
+	supportedDistros = []string{
```

Comment:
This is the home of the supportedDistro by test-infra (ie: the distro for whom we provide prebuilt artifacts). Side effect: we only build configs for supported distros!

Reply:
I think it makes sense @FedeDP!